packages/jingoo/jingoo.1.2.18/opam

```diff
@@ -0,0 +1,15 @@
+opam-version: "1.2"
+maintainer: "Masaki WATANABE <lambda.watanabe[at]gmail.com>"
+authors: "Masaki WATANABE <lambda.watanabe[at]gmail.com>"
+homepage: "https://github.com/tategakibunko/jingoo"
+bug-reports: "https://github.com/tategakibunko/jingoo/issues"
+dev-repo: "git@github.com:tategakibunko/jingoo.git"
+license: "BSD-3"
+build: [
+  [make] {ocaml-native}
+  [make "byte"] {!ocaml-native}
+]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "jingoo"]
+depends: ["ocamlfind" "pcre" "uutf" "ounit"]
```

Comment:
Could you flag `ocamlfind` as a `{build}` dependency and `ounit` as a `{test}` dependency?
I mean, if they are of course.